src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
 
 const ChatEntry = (props) => {
+
+  const likeHearts = props.liked ? '❤️' : '🤍'
+  const msgOrientation = props.id % 2 == 0 ? 'chat-entry local' : 'chat-entry remote';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={msgOrientation}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time">
+          <TimeStamp time={props.timeStamp} />
+        </p>
+        <button className="like" onClick={() => props.toggleLike(props.id)} >{likeHearts}</button>
       </section>
     </div>
   );
 };
 
+
+
 ChatEntry.propTypes = {
```

Comment:
In here we want to make sure that we are passing in all other props, in `chatEntry` you made use of the `id` and `liked` props. So they would need to be in here as well.